tests/PHPUnit/Framework/TestingEnvironmentManipulator.php

```diff
@@ -223,7 +223,7 @@ public function getExtraDefinitions()
                             });
                         }
                     })
-                ]])
+                    ]])
```

Comment:
Seems like PHPCS would expect one `]` to be on the previous line here to not mess up the indentation here.